internal/datastore/spanner/migrations/zz_migration.0010_add_transaction_metadata_table.go

```diff
@@ -0,0 +1,34 @@
+package migrations
+
+import (
+	"context"
+
+	"cloud.google.com/go/spanner/admin/database/apiv1/databasepb"
+)
+
+const (
+	addTransactionMetadataTable = `CREATE TABLE transaction_metadata (
+			transaction_tag STRING(MAX) NOT NULL,
+			created_at TIMESTAMP DEFAULT (CURRENT_TIMESTAMP()),
+			metadata JSON
+		) PRIMARY KEY (transaction_tag),
+		ROW DELETION POLICY (OLDER_THAN(created_at, INTERVAL 2 DAY))
```

Comment:
same feedback as with CRDB around TTL configuration.  Does `interval 2 day` means "GC every second day"?

Reply:
It means after 2 days